src/components/Image/Image.tsx

```diff
@@ -37,11 +37,11 @@ const DeviceSpecificFragment = ({
             <source
                 srcSet={checkWebP(src)}
                 type="image/webp"
-                media={`(max-width: ${BREAKPOINTS[breakpoint]}px)`}
+                media={`(max-width: ${breakpoint})`}
```

Comment:
Let's put `px` here inside and remove extra parenthesis

media={`(max-width: ${breakpoint}px`}
![telegram-cloud-photo-size-2-5217477476513602623-x](https://github.com/gravity-ui/page-constructor/assets/22456330/be72f000-7dee-4322-bd31-0550bcffa102)

Reply:
Sure